docs/application/native/guides/machine-learning/machine-learning-service.md

```diff
@@ -0,0 +1,75 @@
+# Machine Learning Service
+
+The Machine Learning Service API provides utility interfaces for AI application developers.
+
+These functions allow the following operations with NNStreamer:
+- Set and get the pipeline description with a name, which is a unique key string of applicable service.
+- Delete the pipeline description.
+
+## Prerequisites
+
+To enable your application to use Machine Learning Service API:
```

Comment:
Lists should be introduced with a complete sentence. Ref: https://developers.google.com/style/lists#introductory-sentences-for-lists
```suggestion
Follow the steps below to enable your application to use Machine Learning Service API:
```